app/human-detector-app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "1026298349399",
+    "project_id": "eyespy-27808",
+    "storage_bucket": "eyespy-27808.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:1026298349399:android:d402344472a857ccb3e012",
+        "android_client_info": {
+          "package_name": "com.eyespycamera.humandetectorapp"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "1026298349399-b3v9e4e7gm5bbpqrj7036bpgmocmaqlv.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyCItm2Z_4B3Fo0NNHSVWIQL1l4rtGOb8xM"
```

Comment:
:eyes: I think this is something we need to keep secret

Reply:
OH SHOOT LMAO

Reply:
> eyes I think this is something we need to keep secret

thank god its easy to remake ill get rid of it and just keep note to add it eventually

Reply:
Took it out for now and will add a new one when we get there.  This is mostly for android apps when we get it on the play store (i believe) so we don't need this now